modules/src/weather.py

```diff
@@ -0,0 +1,16 @@
+import re
+import requests
+
+def process(input, entities=None):
+    output = {}
+
+    try:
+        city = entities['city'][0]['value']
+        r = requests.get('http://api.openweathermap.org/data/2.5/weather?q=' + city + '&appid=adee94563461cdf1dcfe25956aea10ae')
+        data = r.json()
+        output['input'] = input
+        output['output'] = 'Current weather: ' + data['main.temp']
```

Comment:
access it as `data['main']['temp']`
also output `data['weather'][0]['description']`
and "the OpenWeatherMap name must be mentioned as a weather source in a visible part of the application." see [here](http://openweathermap.org/price)